router/config.go

```diff
@@ -664,22 +666,30 @@ func LoadConfig(path string) (*Config, error) {
 				}
 			}
 			if value, found := healthChecksMap["linkCheck"]; found {
-				if boltMap, ok := value.(map[interface{}]interface{}); ok {
-					if value, found := boltMap["interval"]; found {
+				if checkMap, ok := value.(map[interface{}]interface{}); ok {
+					if value, found := checkMap["interval"]; found {
 						if val, err := time.ParseDuration(fmt.Sprintf("%v", value)); err == nil {
 							cfg.HealthChecks.LinkCheck.Interval = val
 						} else {
 							return nil, errors.Wrapf(err, "failed to parse healthChecks.linkCheck.interval value '%v", value)
 						}
 					}
-					if value, found := boltMap["minLinks"]; found {
+					if value, found := checkMap["minLinks"]; found {
 						if val, ok := value.(int); ok {
 							cfg.HealthChecks.LinkCheck.MinLinks = val
 						} else {
 							return nil, errors.Wrapf(err, "invalid value [%v] for healthChecks.linkCheck.minLinks", value)
 						}
 					}
 
+					if value, found := checkMap["initialDelay"]; found {
+						if val, err := time.ParseDuration(fmt.Sprintf("%v", value)); err == nil {
+							cfg.HealthChecks.LinkCheck.InitialDelay = val
```

Comment:
accompanying doc/examples are updated elsewhere i take it, right?

Reply:
Yes, examples are in ziti